kythe/cxx/indexer/cxx/testdata/template/template_class_inst_explicit.cc

```diff
@@ -4,6 +4,8 @@ using ExternalDef = int;
 template <typename T> struct C {
   using X = ExternalDef;
 };
+//- @C defines/binding CInstantiation
+//- @C ref CInstantiation
 template struct C<int>;
 //- ExplicitC.node/kind record
```

Comment:
Any relationship (or non-relationship) to assert between `ExplicitC` and `CInstantiation` ? What about to `TemplateC`?

Reply:
Fixed (they're the same nodes).  The relationship to `TemplateC` is covered in other tests.